src/scss/atomic/molecules/_cta-block.scss

```diff
@@ -1,79 +1,52 @@
-.cta__block{
-    .button.button--primary + .link.link--large{
-        margin: $buildingUnit * 2 0 0 0;
-        transform: none;
-    }
-
-    .button .link__underline,
-    .button .link__underline:hover {
-        transform: none;
-        background: none;
-    }
+.cta__block {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 0;
 
-    @include min-mobile-xl {
-        .button.button--primary{
-            margin-right: $buildingUnit * 4;
-        }
-        .button.button--primary ~ .link.link--large{
-            display: inline-block;
-            margin: $buildingUnit 0 0 0;
+    .hero & {
+        align-items: flex-start;
+        @include min-tablet() {
+            align-items: center;
         }
     }
 
+    > a {
```

Comment:
You should target .cta__link, element references will cause styles have unwanted inheritance in other anchors. That's why we use bem.

Reply:
Fixed